### PR TITLE
[vulkan] Fixup width of draw commands.

### PR DESCRIPTION
**Pull request by dj2**
The draw command expects the width and heigth passed into the Draw Rect
command to be the full width/height we want to draw. Not a scale of -1
to 1. This means that we would only draw one corner of the screen
instead of the entire screen.

### CODE REVIEW

**Comment by dj2** [comment]
@jaebaek can you please take a look? This works of me, but I may have failed to understand how Vulkan was working here.

File: src/vulkan/engine_vulkan.cc [hunk 1]
```diff
@@ -324,13 +324,15 @@ Result EngineVulkan::DoDrawRect(const DrawRectCommand* command) {
   float y = command->GetY();
   float width = command->GetWidth();
   float height = command->GetHeight();
+
```
[the remark this answers is not in the document]
Comment (jaebaek) [reply src/vulkan/engine_vulkan.cc]: I recommend quickly running [gapid](https://github.com/google/gapid/releases) to check what is the result rectangle.

[the remark this answers is not in the document]
Comment (jaebaek) [reply src/vulkan/engine_vulkan.cc]: I believe this works fine:

```
  float x = command->GetX();
  float y = command->GetY();
  float width = command->GetWidth();
  float height = command->GetHeight();
  if (command->IsOrtho()) {
    const float frame_width = static_cast<float>(graphics->GetWidth());
    const float frame_height = static_cast<float>(graphics->GetHeight());
    x = ((x / frame_width) * 2.0f) - 1.0f;
    y = ((y / frame_height) * 2.0f) - 1.0f;
    width = (width / frame_width) * 2.0f;
    height = (height / frame_height) * 2.0f;
  }
```

File: tests/cases/draw_rect.vkscript [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#version 430
+
+layout(location = 0) out vec4 final_color;
+
+void main() {
+  final_color = vec4(1, 0, 0, 1);
+}
+
+[test]
+draw rect -1 -1 1 1
```
Comment (jaebaek) [inline tests/cases/draw_rect.vkscript]: This must be `draw rect -1 -1 2 2`

**Comment by dj2** [comment]
Fixes done. PTAL.

**Pull request merged**